src/file_resolver.rs

```diff
@@ -139,7 +139,8 @@ impl FileResolver {
             for (import_no, import) in self.import_paths.iter().enumerate() {
                 if let (Some(mapping), import_path) = import {
                     if first_part == mapping {
-                        // match!
+                        // clippy 1.66 complains that & in &relpath is unnecessary, but we can't remove it due to the loop
+                        #[allow(clippy::needless_borrow)]
```

Comment:
Unnecessary as can do the following instead:

```rust
            let relpath: &PathBuf = &iter.collect();

            for (import_no, import) in self.import_paths.iter().enumerate() {
                if let (Some(mapping), import_path) = import {
                    if first_part == mapping {
                        if let Ok(full_path) = import_path.join(relpath).canonicalize() {
```

Reply:
Brilliant thanks @xermicus